third_party/binutils/test_decenc/drdecode_decenc_x86_64.expect

```diff
@@ -5036,47 +5194,47 @@ test_x86_64_s:
  00 00 07
  c4 43 1d 4b b7 99 00 vblendvpd %ymm12, 0x99(%r15), %ymm8, %ymm14
  00 00 80
- c5 f8 ae 15 99 00 00 vldmxcsr <rel> 0x0000000010005eaf
+ c5 f8 ae 15 99 00 00 vldmxcsr <rel> 0x00000000100061d9
```

Comment:
(I assume these are just from shifting the PC base of the rip-rel addr due to the instrs above inserted in the middle.)

Reply:
Yes, that's right.